examples/userapi/07_sparse_operations.ipynb

```diff
@@ -0,0 +1,889 @@
+{
```

Comment:
<p>Now that it is larger, it makes more sense..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>